Wikipedia/Code/NSDictionary+WMFCommonParams.m

```diff
@@ -25,7 +25,7 @@ + (instancetype)wmf_titlePreviewRequestParametersWithExtractLength:(NSUInteger)e
                                          //@"any", @"pilicense",
                                          imageWidth, @"pithumbsize",
                                          // revision
-                                         @(1), @"rrvlimit",
+                                         //@(1), @"rrvlimit",
```

Comment:
I commented this out because it was causing `"Unrecognized parameter: rrvlimit."` (on generated API urls like [this](https://en.wikipedia.org/w/api.php?prop=description%7Cpageimages%7Cpageprops%7Crevisions%7Cextracts%7Ccoordinates&ppprop=ns%7Cdisplaytitle&rrvlimit=1&piprop=thumbnail&pithumbsize=640&titles=Truxillic%20acid&exchars=525&pilimit=1&rvprop=ids&continue=&exintro=1&action=query&format=json&exlimit=1&explaintext=)), and the fetcher code treated its presence as a complete-failure rather than just a warning. Then I saw an earlier commit (6daf9e58a976e07b4f1313ddace8f2db019caba9) which had commented out the parameter elsewhere, and so...

I could probably remove this from the pull request, given that 088dbc88bf76baea69f274df3b7feedf28740b65 removed the path that fetched the article preview anyway.

Reply:
It's OK either way,`WMFArticlePreviewFetcher` will be removed entirely, it's only used in one other place. I added it to the list for https://phabricator.wikimedia.org/T217411